src/StripeTests/Services/Issuing/Cards/IssuingCardServiceTest.cs

```diff
@@ -6,7 +6,7 @@ namespace StripeTests
     using Stripe.Issuing;
     using Xunit;
 
-    public class CardServiceTest : BaseStripeTest
+    public class IssuingCardServiceTest : BaseStripeTest
```

Comment:
Flagging that there's a name conflict in the test suite as we already have `CardServiceTest`. For now, I just prefixed that one and the DisputeServiceTest one to avoid conflicts until we find a better solution

Reply:
We can probably reuse the namespacing in tests, e.g. we'd have `StripeTests.CardServiceTest` and `StripeTests.Issuing.CardServiceTest`.